utils/language_extension.ml

```diff
@@ -27,13 +29,15 @@ let to_string = function
   | Include_functor -> "include_functor"
   | Polymorphic_parameters -> "polymorphic_parameters"
   | Immutable_arrays -> "immutable_arrays_experimental"
+  | Strengthen -> "strengthen"
```

Comment:
:bike::hut:: What about `Module_strengthening`?  I find "strengthen" lacking too much context.

Reply:
(I spent several seconds wondering what "motorbike-hut" meant, as that's what originally parsed.)

Yes, you're right.

Reply:
Actually, I think @rleshchinskiy gets the final call here. (The pedant in me prefers `module_type_strengthening`, though that's really too long.) But I've made this change in this patch.